### ISSUE TITLE
remove all "foos" and "bars"

### ISSUE THREAD

**Issue by timruffles** [BonsaiDen/JavaScript-Garden#261]
foos and bars are horrible in example code as it gives no context on _why_ you'd do something.

**Comment by chrisranderson** [comment]
Agreed. They could at least be named "some" and "another" so it's easier to remember which function or variable is which. "first" and "second" could work too.

Foo and bar just add mental strain to the examples:
var foo = {name: 'kitten'}
foo.name; // kitten
foo['name']; // kitten

var get = 'name';
foo[get]; // kitten

foo.1234; // SyntaxError
foo['1234']; // works

Why would you ever name foo kitten? It makes it harder to remember what's going on. Easier to use var person, and person.name. If you keep to a theme throughout the document I bet it would erase a lot of the foo bar overhead.

**Comment by hugotox** [comment]
foo and bar are specially confusing for non english speakers (like me) because we can't translate/understand that words

**Comment by timruffles** [comment]
Still a long way to go:

```sh
$ ack -hc '\b(foo|bar|baz|qux)\b' doc/en/
164
```

**Comment by roccomuso** [comment]
Totally Agree.

**Comment by timruffles** [comment]
PRs would be gratefully accepted on this one. Puns win extra points.

**Comment by siddharthkp** [comment]
Here's my tiny contribution: https://github.com/siddharthkp/JavaScript-Garden/pull/1

**Comment by siddharthkp** [comment]
My tiny contribution: https://github.com/BonsaiDen/JavaScript-Garden/pull/373

**Comment by alieldeba** [comment]
agreed with that
